src/specifications/profile-resolution/readme.md

```diff
@@ -0,0 +1,29 @@
+## Editing this specification
+
+Find an oXygen project file inside the file `profile-resolution-spec.xpr`.
+
+Features of the setup:
+
+* XML authoring using raw tags or alternatively, in a structured editing environment with outlining, layout, formatting and UI widgets
+* One-button production of a self-contained HTML file for preview in a browser
+* Another button for production of a version for the web site (targeting NIST/USWDS/Hugo - in development
+* Schematron support for ad-hoc rules
+* Lightweight tagging is an HTML core plus a little healthy XML goodness for a few specification-oriented semantics
```

Comment:
The meaning of this line isn't very clear. Is more about the implementation than the capability?

Reply:
Superseded